app/src/main/java/com/realityexpander/tasky/core/domain/typeParcelers/LocalDateTimeParceler.kt

```diff
@@ -0,0 +1,16 @@
+package com.realityexpander.tasky.core.domain.typeParcelers
+
+import android.os.Parcel
+import kotlinx.parcelize.Parceler
+import java.time.LocalDateTime
+
+object LocalDateTimeParceler : Parceler<LocalDateTime> {
+    override fun create(parcel: Parcel): LocalDateTime {
+        return LocalDateTime.parse(parcel.readString())
+    }
+
+    override fun LocalDateTime.write(parcel: Parcel, flags: Int) {
+        parcel.writeString(this.toString())
+    }
+
+}
```

Comment:
Can't you just use the `@Parcelize` annotation instead of this?

Reply:
I dont think the Parcelize will convert the custom types of `LocalDateTime` and `ZonedDateTime`, so I thought I needed to use a custom `@Parceler`. I will attempt to use it without the custom parceler and see if the `@Parcelize` does this for me.

Reply:
Turns out it does it automatically, so i will remove this.